src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities.mdx

```diff
@@ -39,7 +42,7 @@ Our pre-built roles have various groupings of capabilities. How our pre-built ro
 
 ## Capability definitions [#capability-definitions]
 
-Here's a screenshot of the capabilities available in the capabilities UI. These are only a subset of everything you can do in New Relic and represent the specific capabilities we think might be valuable for enabling or disabling for a custom role. 
+Here's a screenshot of the capabilities available in the capabilities UI. These are only a subset of everything you can do in New Relic and represent the specific capabilities we believe are valuable for enabling or disabling for a custom role. 
```

Comment:
Consider moving these sentences to a figCaption of the image, and then replace it with a simple intro + instructions on where to find capabilities in the UI.